lib/repository/sembast/sembast_repository.dart

```diff
@@ -518,6 +518,9 @@ class SembastRepository extends Repository {
       case PodcastEpisodeFilter.notPlayed:
         episodeFilter = Filter.and([Filter.equals('pguid', pguid), Filter.equals('played', 'false')]);
         break;
+      case PodcastEpisodeFilter.downloaded:
```

Comment:
This is the correct place to add the new filter, but the `downloaded` value is wrong. Take a look at the `findDownloads()` method for a pointer on the correct filter.